### ISSUE TITLE
http://www.ruby-rbot.org/ seems to be gone.

### ISSUE THREAD

**Issue by antdude** [ruby-rbot/rbot#89]
http://www.ruby-rbot.org/ doesn't connect and ping. :(

$ ping ruby-rbot.org
ping: ruby-rbot.org: Name or service not known

**Comment by antdude** [comment]
And now, it goes to https://www.livewallpapers.com/tech-wallpapers/robot-wallpapers. :(

**Comment by henk84** [comment]
Yes, correct, the domain has expired and intentionally been let expire because of the state of development (mostly stale) and the domain costing money. At least AFAIU.
We should clean up references to it in the code and replace them with the new website https://ruby-rbot.github.io/

**Comment by antdude** [comment]
Thanks. So, does this mean this old rbot project back to active status now?

**Comment by henk84** [comment]
> Thanks. So, does this mean this old rbot project back to active status now?

There currently is some activity but I can’t make any promises where this will go or how long it can be kept up (;
